draft-ietf-rats-corim.md

```diff
@@ -2319,7 +2319,36 @@ groups to use this information as they see fit".
 
 # Security and Privacy Considerations {#sec-sec}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/11
+Evidence appraisal is at the core of any RATS protocol flow, mediating all interactions between Attesters and their Relying Parties.
+The Verifier is effectively part of the Attesters' and Relying Parties' trusted computing base (TCB).
```

Comment:
```suggestion
The Verifier is effectively part of the Attesters' and Relying Parties' Trusted Computing Base (TCB).
```

Reply:
I don't think capitalisation is required.  See for example https://en.wikipedia.org/wiki/Trusted_computing_base

Reply:
Yes, strictly not required, but see how it renders nicely: https://confidentialcomputing.io/about/